octoprint_octorelay/__init__.py

```diff
@@ -126,18 +126,24 @@ def handle_get_status_command(self, index: str):
         self._logger.debug(f"Responding to {GET_STATUS_COMMAND} command: {is_closed}")
         return flask.jsonify(status=is_closed)
 
-    def handle_update_command(self, index: str):
-        self._logger.debug(f"Requested to switch the relay {index}")
+    def handle_update_command(self, data ):
```

Comment:
I'd like handlers to operate the parsed data (move `data` processing to `on_api_command` method)

```suggestion
    def handle_update_command(self, index: str, target: Optional[bool]):
```